ScrapPackedLibrary/DataTypes/PackedMetaData.cs

```diff
@@ -1,36 +1,37 @@
 ﻿using System;
 using System.Collections.Generic;
-using System.Text;
 
 namespace ch.romibi.Scrap.Packed.PackerLib.DataTypes
 {
     public class PackedMetaData
     {
-        private const UInt32 DATA_LENGTH_STATIC = 12; // 4 bytes each: "PFBK", version (all 0s), number of files
-
-        public const string fileHeader = "BFPK";
-        public UInt32 packedVersion { get; set; } // always 0 (not sure if really a version number)
-        public List<PackedFileIndexData> fileList { get; set; }
-        public Dictionary<string, PackedFileIndexData> fileByPath { get; set; }
+        // Fields
+        public const string FileHeader = "BFPK";
+        public uint PackedVersion { get; set; } // always 0 (not sure if really a version number)

```

Comment:
**Regarding all changes from `UInt32` to `uint`:**
Will `uint` always for 100% sure on all architectures be an equivalent datatype to `UInt32`?
Because it **HAS** to be 32bit. (Range 0 to 4,294,967,295)

Reply:
No, it is not 32-bits in all systems. 

Missconfigured .editorconf + "code cleaner" button.